frontend/src/components/pages/details/components/DetailsSection/DetailsSection.tsx

```diff
@@ -19,12 +17,13 @@ export const DetailsSection: React.FC<DetailsSectionProps> = ({
 }) => {
   return (
     <div className={cn('scroll-mt-20 desktop:scroll-mt-30', className)} id={htmlId}>
-      <ScrollContainer
+      <div
         id="details_section"
         className={`flex flex-col
           pt-6 desktop:pt-12
           pb-3 desktop:pb-6
           mb-3 desktop:mb-6
+          max-w-full overflow-x-auto
```

Comment:
This div will render with a class equal to `flex flex-col\n          pt-6 desktop:pt-12\n          pb-3 desktop:pb-6\n etc...` which isn't... _great_, I don't know if this is done in a lot of other places but maybe there's a better way? Using some tailwind helpers magic?